scripts/add_link_slope_to_network.py

```diff
@@ -3,9 +3,11 @@
 import os
 import json
 
+import genet
 from genet import read_matsim
 from genet.utils.persistence import ensure_dir
-import genet.outputs_handler.sanitiser as sanitiser
+import genet.output.sanitiser as sanitiser
+import genet.utils.elevation as elevation
 
 if __name__ == '__main__':
     arg_parser = argparse.ArgumentParser(description='Add elevation data to network nodes and validate it')
```

Comment:
add something about computing link slopes to description and that you can do both or just one of them